  <!-- Imprint Section -->
  <section id="publications">
    <div class="container">
      <h4>Publications</h4>
      <div id="overview">
        <p>
          35 journal articles<br>
          2 book chapters<br>
          h-index:
          <a href="https://scholar.google.de/citations?hl=de&user=uzZoRSsAAAAJ">18</a><br>
          Last updated: 2021-04-01
        </p>
      </div>
      <h5>Journal Articles</h5>
      <div id="articles">
        <table class="publication-table">
          <caption>2021</caption>
          <tbody>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1073/pnas.2013043118">The mutational load in natural populations is significantly affected by high primary rates of retroposition</a></span>
                <br>Zhang W, Xie C, <b>Ullrich K</b>, Zhang YE, Tautz D
                <br><i>Proceedings of the National Academy of Sciences </i><b>118 (6)</b>, e2013043118.
              </td>
            </tr>
          </tbody>
        </table>
        <table class="publication-table">
          <caption>2020</caption>
          <tbody>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1186/s12862-020-01624-5">The amylase gene cluster in house mice (Mus musculus) was subject to repeated introgression including the rescue of a pseudogene</a></span>
                <br>Linnenbrink M, <b>Ullrich KK</b>, McConnell E, Tautz D
                <br><i>BMC evolutionary biology </i><b>20</b>, 1-11.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.21105/joss.02424">CRBHits: From Conditional Reciprocal Best Hits toCodon Alignments and Ka/Ks in R</a></span>
                <br><b>Ullrich KK</b>
                <br><i>The Journal of Open Source Software </i><b>5 (55)</b>, .
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1371/journal.pcbi.1008354">Dedicated transcriptomics combined with power analysis lead to functional understanding of genes with weak phenotypic changes in knockout lines</a></span>
                <br>Xie C, Bekpen C, Künzel S, Keshavarz M, Krebs-Wheaton R, Skrabar N, <b>Ullrich KK</b>, Zhang W, Tautz D
                <br><i>PLoS computational biology </i><b>16 (11)</b>, e1008354.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1111/tpj.14607">PEATmoss (Physcomitrella Expression Atlas Tool): a unified gene expression atlas for the model plant Physcomitrella patens</a></span>
                <br>Fernandez‐Pozo N, Haas FB, Meyberg R, <b>Ullrich KK</b>, Hiss M, Perroud PF, Hanke S, Kratz V, Powell AF, Vesty EF, Daum CG, Zane M, Lipzen A, Sreedasyam A, Grimwood J, Coates JC, Barry K, Schmutz J, Mueller LA, Rensing SA
                <br><i>The Plant Journal </i><b>102 (1)</b>, 165-177.
              </td>
            </tr>
          </tbody>
        </table>
        <table class="publication-table">
          <caption>2019</caption>
          <tbody>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1186/s12864-019-5452-4">Usability of reference-free transcriptome assemblies for detection of differential expression: a case study on Aethionema arabicum dimorphic seeds</a></span>
                <br>Wilhelmsson PKI, Chandler JO, Fernandez-Pozo N, Graeber K, <b>Ullrich KK</b>, Arshad W, Khan S, Hofberger JA, Buchta K, Edger PP, Pires JC, Schranz ME, Leubner-Metzger G, Rensing SA
                <br><i>BMC genomics </i><b>20 (1)</b>, 1-19.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1186/s12864-019-5452-4">Genomic Structure of Hstx2 Modifier of Prdm9-Dependent Hybrid Male Sterility in Mice</a></span>
                <br>Lustyk D, Kinský S, <b>Ullrich KK</b>, Yancoskie M, Kašíková L, Gergelits V, Sedlacek R, Chan YF, Odenthal-Hesse L, Forejt J, Jansa P
                <br><i>Genetics </i><b>213 (3)</b>, 1047-1063.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1038/s41586-019-1693-2">One thousand plant transcriptomes and the phylogenomics of green plants</a></span>
                <br>Leebens-Mack JH, Barker MS, Carpenter EJ, Deyholos MK, Gitzendanner MA, Graham SW, Grosse I, Li Z, Melkonian M, Mirarab S, Porsch M, Quint M, Rensing SA, Soltis DE, Soltis PS, Stevenson DW, <b>Ullrich KK</b>, Wickett NJ, DeGironimo L, Edger PP, Jordon-Thaden IE, Joya S, Liu T, Melkonian B, Miles NW, Pokorny L, Quigley C, Thomas P, Villarreal JC, Augustin MM, Barrett MD, Baucom RS, Beerling DJ, Benstein RM, Biffin E, Brockington SF, Burge DO, Burris JN, Burris KP, Burtet-Sarramegna V, Caicedo AL, Cannon SB, Cebi Z, Chang Y, Chater C, Cheeseman JM, Chen T, Clarke ND, Clayton H, Covshoff S, Crandall-Stotler BJ, Cross H, dePamphilis CW, Der JP, Determann R, Dickson RC, Stilio VSD, Ellis S, Fast E, Feja N, Field KJ, Filatov DA, Finnegan PM, Floyd SK, Fogliani B, Garcia N, Gateble G, Godden GT, Goh FQY, Greiner S, Harkess A, Heaney JM, Helliwell KE, Heyduk K, Hibberd JM, Hodel RGJ, Hollingsworth PM, Johnson MTJ, Jost R, Joyce B, Kapralov MV, Kazamia E, Kellogg EA, Koch MA, Von Konrat M, Konyves K, Kutchan TM, Lam V, Larsson A, Leitch AR, Lentz R, Li FW, Lowe AJ, Ludwig M, Manos PS, Mavrodiev E, McCormick MK, McKain M, McLellan T, McNeal JR, Miller RE, Nelson MN, Peng Y, Ralph P, Real D, Riggins CW, Ruhsam M, Sage RF, Sakai AK, Scascitella M, Schilling EE, Schlosser EM, Sederoff H, Servick S, Sessa EB, Shaw AJ, Shaw SW, Sigel EM, Skema C, Smith AG, Smithson A, Stewart CN,  Stinchcombe JR, Szovenyi P, Tate JA, Tiebel H, Trapnell D, Villegente M, Wang CN, Weller SG, Wenzel M, Weststrand S, Westwood JH, Whigham DF, Wu S, Wulff AS, Yang Y, Zhu D, Zhuang C, Zuidof J, Chase MW, Pires JC, Rothfels CJ, Yu J, Chen C, Chen L, Cheng S, Li J, Li R, Li X, Lu H, Ou Y, Sun X, Tan X, Tang J, Tian Z, Wang F, Wang J, Wei X, Xu X, Yan Z, Yang F, Zhong X, Zhou F, Zhu Y, Zhang Y, Ayyampalayam S, Barkman TJ, Nguyen N, Matasci N, Nelson DR, Sayyari E, Wafula EK, Walls RL, Warnow T, An H, Arrigo N, Baniaga AE, Galuska S, Jorgensen SA, Kidder TI, Kong H, Lu-Irving P, Marx HE, Qi X, Reardon CR, Sutherland BL, Tiley GP, Welles SR, Yu R, Zhan S, Gramzow L, Theißen G and Wong GKS
                <br><i>Nature </i><b>574</b>, 679–685.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1093/gigascience/giz126">Access to RNA-sequencing data from 1,173 plant species: The 1000 Plant transcriptomes initiative (1KP)</a></span>
                <br>Carpenter EJ, Matasci N, Ayyampalayam S, Wu S, Sun J, Yu J, Vieira FRJ, Bowler C, Dorrell RG, Gitzendanner MA, Li L, Du W, <b>Ullrich KK</b>, Wickett NJ, Barkmann TJ, Barker MS, Leebens-Mack JH, Wong GKS
                <br><i>GigaScience </i><b>8 (10)</b>, giz126.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.7554/eLife.44392.001">A de novo evolved gene in the house mouse regulates female pregnancy cycles</a></span>
                <br>Xie C, Bekpen C, Künzel S, Keshavarz M, Krebs-Wheaton R, Skrabar N, <b>Ullrich KK</b>, Tautz D
                <br><i>Elife </i><b>8</b>, e44392.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1534/g3.119.400083">A Reference Genome Sequence for the European Silver Fir (Abies alba Mill.): A Community-Generated Genomic Resource</a></span>
                <br>Mosca E, Cruz F, Gómez-Garrido J, Bianco L, Rellstab C, Brodbeck S, Csilléry K, Fady B, Fladung M, Fussi B, Gömöry D, González-Martínez SC, Grivet D, Gut M, Hansen OK, Heer , Kaya Z, Krutovsky KV, Kersten B, Liepelt S, Opgenoorth L, Sperisen C, <b>Ullrich KK</b>, Vendramin GG, Westergren M, Ziegenhagen B,  Alioto T, Gugerli F, Heinze B, Höhn M, Troggio M, Neale DB
                <br><i>G3: Genes, Genomes, Genetics </i><b>9 (7)</b>, 2039-2049.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1093/jxb/erz146">Aethionema arabicum: a novel model plant to study the light control of seed germination</a></span>
                <br>Mérai Z, Graeber K, Wilhelmsson PKI, <b>Ullrich KK</b>, Arshad W, Grosche C, Tarkowská D, Turečková V, Strnad M, Rensing SA, Leubner-Metzger G, Scheid OM
                <br><i>Journal of experimental botany </i><b>70 (12)</b>, 3313-3328.
              </td>
            </tr>
          </tbody>
        </table>
        <table class="publication-table">
          <caption>2018</caption>
         <tbody>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1038/s41559-018-0639-7">Incipient de novo genes can evolve from frozen accidents that escaped rapid transcript turnover</a></span>
                <br>Schmitz JF, <b>Ullrich KK</b>, Bornberg-Bauer E
                <br><i>Nature ecology & evolution </i><b>2 (10)</b>, 1626-1632.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1002/ece3.4374">Detection of somatic epigenetic variation in Norway spruce via targeted bisulfite sequencing</a></span>
                <br>Heer K, <b>Ullrich KK</b>, Hiss M, Liepelt S, Brüning RS, Zhou J, Opgenoorth L, Rensing SA
                <br><i>Ecology and evolution </i><b>8 (19)</b>, 9672-9682.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1007/s00709-018-1232-2">The effect of auxin (indole-3-acetic acid) on the growth rate and tropism of the sporangiophore of Phycomyces blakesleeanus and identification of auxin-related genes</a></span>
                <br>Živanović BD, <b>Ullrich KK</b>, Steffens B, Spasić SZ, Galland P
                <br><i>Protoplasma </i><b>255 (5)</b>, 1331-1347.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1016/j.cell.2018.06.033">The Chara genome: secondary complexity and implications for plant terrestrialization</a></span>
                <br>Nishiyama T, Sakayama H, De Vries J, Buschmann H, Saint-Marcoux D, <b>Ullrich KK</b>, Haas FB, Vanderstraeten L, Becker D, Lang D, Vosolsobě S, Rombauts S, Wilhelmsson PKI, Janitza P, Kern R, Heyl A, Rümpler F, Villalobos LIAC, Clay JM, Skokan R, Toyoda A, Suzuki Y, Kagoshima H, Schijlen E, Tajeshwar N, Catarino B, Hetherington AJ, Saltykova A, Bonnot C, Breuninger H, Symeonidi A, Radhakrishnan GV, Van Nieuwerburgh F, Deforce D, Chang C, Karol KG, Hedrich R, Ulvskov P, Glöckner G, Delwiche CF, Petrášek J, Van de Peer Y, Friml J, Beilby M, Dolan L, Kohara Y, Sugano S, Fujiyama A, Delaux PM, Quint M, Theißen G, Hagemann M, Harholt J, Dunand C, Zachgo S, Langdale J, Maumus F, Van Der Straeten D, Gould SB, Rensing SA
                <br><i>Cell </i><b>174 (2)</b>, 448-464.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1111/tpj.13940">The Physcomitrella patens gene atlas project: large‐scale RNA‐seq based expression data</a></span>
                <br>Perroud PF, Haas FB, Hiss M, <b>Ullrich KK</b>, Alboresi A, Amirebrahimi M, Barry K, Bassi R, Bonhomme S, Chen H, Coates JC, Fujita T, Guyon‐Debast A, Lang D, Lin J, Lipzen A, Nogué F, Oliver MJ, de León IP, Quatrano RS, Rameau C, Reiss B, Reski R, Ricca M, Saidi Y, Sun N, Szövényi P, Sreedasyam A, Grimwood J, Stacey G, Schmutz J, Rensing SA
                <br><i>The Plant Journal </i><b>95 (1)</b>, 168-182.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1111/tpj.13801">The Physcomitrella patens chromosome‐scale assembly reveals moss genome structure and evolution</a></span>
                <br>Lang D, <b>Ullrich KK</b>, Murat F, Fuchs J, Jenkins J, Haas FB, Piednoel M, Gundlach H, Van Bel M, Meyberg R, Vives C, Morata J, Symeonidi A, Hiss M, Muchero W, Kamisugi Y, Saleh O, Blanc G, Decker EL, van Gessel N, Grimwood J, Hayes RD, Graham SW, Gunter LE, McDaniel SF, Hoernstein SNW, Larsson A, Li FW, Perroud PF, Phillips J, Ranjan P, Rokshar DS, Rothfels CJ, Schneider L, Shu S, Stevenson DW, Thümmler F, Tillich M, Aguilar JCV, Widiez T, Wong GKS, Wymore A, Zhang Y, Zimmer AD, Quatrano RS, Mayer KFX, Goodstein D, Casacuberta JM, Vandepoele K, Reski R, Cuming AC, Tuskan GA, Maumus F, Salse J, Schmutz J, Rensing SA
                <br><i>The Plant Journal </i><b>93 (3)</b>, 515-533.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1111/nph.14860">The sphagnome project: enabling ecological and evolutionary insights through a genus‐level sequencing project</a></span>
                <br>Weston DJ, Turetsky MR, Johnson MG, Granath G, Lindo Z, Belyea LR, Rice SK, Hanson DT, Engelhardt KAM, Schmutz J, Dorrepaal E, Euskirchen ES, Stenøien HK, Szövényi P, Jackson M, Piatkowski BT, Muchero W, Norby RJ, Kostka JE, Glass JB, Rydin H, Limpens J, Tuittila ES, <b>Ullrich KK</b>, Carrell A, Benscoter BW, Chen JG, Oke TA, Nilsson MB, Ranjan P, Jacobson D, Lilleskov EA, Clymo RS, Shaw AJ
                <br><i>New Phytologist </i><b>217 (1)</b>, 16-25.
              </td>
            </tr>
          </tbody>
        </table>
        <table class="publication-table">
          <caption>2017</caption>
          <tbody>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1111/ele.12858">Ecological plant epigenetics: Evidence from model and non‐model species, and the way forward</a></span>
                <br>Richards CL, Alonso C, Becker C, Bossdorf O, Bucher E, Colomé‐Tatché M, Durka W, Engelhardt J, Gaspar B, Gogol‐Döring A, Grosse I, van Gurp TP, Heer K, Kronholm I, Lampei C, Latzel V, Mirouze M, Opgenoorth L, Paun O, Prohaska SJ, Rensing SA, Stadler PF, Trucchi E, <b>Ullrich K</b>, Verhoeven KJF
                <br><i>Ecology letters </i><b>20 (12)</b>, 1576-1590.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1093/gbe/evx258">Comprehensive genome-wide classification reveals that many plant-specific transcription factors evolved in streptophyte algae</a></span>
                <br>Wilhelmsson PKI, Mühlich C, <b>Ullrich KK</b>, Rensing SA
                <br><i>Genome biology and evolution </i><b>9 (12)</b>, 3384-3397.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.3389/fpls.2017.01842">Combination of the endogenous lhcsr1 promoter and codon usage optimization boosts protein expression in the moss Physcomitrella patens</a></span>
                <br>Hiss M, Schneider L, Grosche C, Barth MA, Neu C, Symeonidi A, <b>Ullrich KK</b>, Perroud PF, Schallenberg-Rüdinger M, Rensing SA
                <br><i>Frontiers in plant science </i><b>8</b>, 1842.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1104/pp.17.00733">Protoplast swelling and hypocotyl growth depend on different auxin signaling pathways</a></span>
                <br>Dahlke RI, Fraas S, <b>Ullrich KK</b>, Heinemann K, Romeiks M, Rickmeyer T, Klebe G, Palme K, Lüthen H, Steffens B
                <br><i>Plant physiology </i><b>175 (2)</b>, 982-994.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1093/gbe/evx098">Selfing in Haploid Plants and Efficacy of Selection: Codon Usage Bias in the Model Moss Physcomitrella patens</a></span>
                <br>Szövényi P, <b>Ullrich KK</b>, Rensing SA, Lang D, van Gessel N, Stenøien HK, Conti E, Reski R
                <br><i>Genome biology and evolution </i><b>9 (6)</b>, 1528-1546.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1111/tpj.13501">Sexual reproduction, sporophyte development and molecular variation in the model moss Physcomitrella patens: introducing the ecotype Reute</a></span>
                <br>Hiss M, Meyberg R, Westermann J, Haas FB, Schneider L, Schallenberg‐Rüdinger M, <b>Ullrich KK</b>, Rensing SA
                <br><i>The Plant Journal </i><b>90 (3)</b>, 606-620.
              </td>
            </tr>
          </tbody>
        </table>
        <table class="publication-table">
          <caption>2016</caption>
          <tbody>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1007/s12686-016-0520-4">Detection of SNPs based on transcriptome sequencing in Norway spruce (Picea abies (L.) Karst)</a></span>
                <br>Heer K, <b>Ullrich KK</b>, Liepelt L, Rensing SA, Zhou J, Ziegenhagen B, Opgenoorth L
                <br><i>Conservation Genetics Resources </i><b>8 (2)</b>, 105-107.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1111/mec.13516">Evidence of divergent selection for drought and cold tolerance at landscape and local scales in Abies alba Mill. in the French Mediterranean Alps</a></span>
                <br>Roschanski AM, Csilléry K, Liepelt S, Oddou‐Muratorio S, Ziegenhagen B, Huard F, <b>Ullrich KK</b>, Postolache D, Vendramin GG, Fady B
                <br><i>Molecular Ecology </i><b>25 (3)</b>, 776-794.
              </td>
            </tr>
          </tbody>
        </table>
        <table class="publication-table">
          <caption>2015</caption>
          <tbody>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1186/s12870-015-0566-6">Natural variants of ELF3 affect thermomorphogenesis by transcriptionally modulating PIF4-dependent auxin response genes</a></span>
                <br>Raschke A, Ibañez C, <b>Ullrich KK</b>, Anwer MU, Becker S, Glöckner A, Trenner J, Denk K, Saal B, Sun X, Ni M, Davis SJ, Delker C, Quint M
                <br><i>BMC plant biology </i><b>15 (1)</b>, 1-10.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1016/j.copbio.2014.11.011">Means to optimize protein expression in transgenic plants</a></span>
                <br><b>Ullrich KK</b>, Hiss M, Rensing SA
                <br><i>Current opinion in biotechnology </i><b>32</b>, 61-67.
              </td>
            </tr>
          </tbody>
        </table>
        <table class="publication-table">
          <caption>2014</caption>
          <tbody>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1186/1471-2229-14-178">Functional analysis of COP1 and SPA orthologs from Physcomitrella and rice during photomorphogenesis of transgenic Arabidopsis reveals distinct evolutionary conservation</a></span>
                <br>Ranjan A, Dickopf S, <b>Ullrich KK</b>, Rensing SA, U Hoecker
                <br><i>BMC plant biology </i><b>14 (1)</b>, 1-15.
              </td>
            </tr>
          </tbody>
        </table>
        <table class="publication-table">
          <caption>2013</caption>
          <tbody>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1371/journal.pone.0078497">Optimized probe masking for comparative transcriptomics of closely related species</a></span>
                <br>Poeschl Y, Delker C, Trenner J, <b>Ullrich KK</b>, Quint M, Grosse I
                <br><i>PloS one </i><b>8 (11)</b>, e78497.
              </td>
            </tr>
          </tbody>
        </table>
        <table class="publication-table">
          <caption>2012</caption>
          <tbody>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.3389/fpls.2012.00271">Toward a comprehensive phylogenetic reconstruction of the evolutionary history of mitogen-activated protein kinases in the plant kingdom</a></span>
                <br>Janitza P, <b>Ullrich KK</b>, Quint M
                <br><i>Frontiers in plant science </i><b>3</b>, 271.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1038/nature11394">A transcriptomic hourglass in plant embryogenesis</a></span>
                <br>Quint M, Drost HG, Gabel A, <b>Ullrich KK</b>, Bönn M, Grosse I
                <br><i>Nature </i><b>490 (7418)</b>, 98-101.
              </td>
            </tr>
          </tbody>
        </table>
        <table class="publication-table">
          <caption>2011</caption>
          <tbody>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1104/pp.110.166579">Molecular evolution and selection patterns of plant F-box proteins with C-terminal kelch repeats</a></span>
                <br>Schumann N, Navarro-Quezada A, <b>Ullrich K</b>, Kuhl C, Quint M
                <br><i>Plant physiology </i><b>155 (2)</b>, 835-850.
              </td>
            </tr>
          </tbody>
        </table>
        <table class="publication-table">
          <caption>2010</caption>
          <tbody>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1105/tpc.110.073957">Natural variation of transcriptional auxin response networks in Arabidopsis thaliana</a></span>
                <br>Delker C, Pöschl Y, Raschke A, <b>Ullrich K</b>, Ettingshausen E, Hauptmann V, Grosse I, Quint M
                <br><i>The Plant Cell </i><b>22 (7)</b>, 2184-2200.
              </td>
            </tr>
            <tr>
              <td><br></td>
            </tr>
          </tbody>
        </table>
      </div>
      <h5>Book Chapters</h5>
      <div id="books">
        <table class="publication-table">
          <tbody>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1016/bs.abr.2016.01.004">The genome of the model moss Physcomitrella patens</a></span>
                <br>Lang D, Van Gessel N, <b>Ullrich KK</b>, Reski R
                <br><i>Advances in botanical research </i><b>78 (2016)</b>, 97-140.
              </td>
            </tr>
            <tr>
              <td style="text-align:left;"><span class="pub-title"><a href="https://doi.org/10.1007/978-1-0716-0199-0_18">Population genomics of the house mouse and the Brown rat</a></span>
                <br><b>Ullrich KK</b>, Tautz D
                <br><i>Statistical Population Genomics </i><b>(2020)</b>, 435-452.
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </section>
